### DISCUSSION TITLE
How to host the  clean architecture template on Debian 10

### DISCUSSION THREAD

**Discussion by 1988gadocansey** [jasontaylordev/CleanArchitecture#842] [General]
I have this environment

Debian 10

Postgres
Ngnix
Dotnet 7.0
Angular15

How do i point my / location since theres a redirection to 44447 before the angular frontend is served?
 
location / {
       root /var/www/www/cleanarchitecture;
       index index.html;
       #try_files $uri $uri/ /index.html;
       }

**Comment by jasontaylordev** [comment]
I can provide support for this but I might be able to point you in the right direction. The template was based on the official Angular + .NET SPA template, and Microsoft has provided some guidance here; https://learn.microsoft.com/en-us/aspnet/core/client-side/spa/angular?view=aspnetcore-7.0&tabs=visual-studio#publish-and-deploy.

Hopefully that will get you started, good luck! 😀
